hal_st/stm32fxxx/SpiMasterStm.cpp

```diff
@@ -95,61 +117,63 @@ namespace hal
     void SpiMasterStm::HandleInterrupt()
     {
         uint32_t status = peripheralSpi[spiInstance]->SR;
-#if !defined(STM32WBA)
+#ifdef SPI_FLAG_RXNE
         if ((status & SPI_FLAG_RXNE) != 0)
 #else
-        if ((status & SPI_FLAG_RXWNE) != 0)
+        if ((status & SPI_IT_RXP) != 0)
```

Comment:
Is this a correct change? status is from the SR register. SPI_FLAG_RXWNE is defined as SPI_SR_RXWNE, so at least that one points to a SPI SR definition. SPI_IT_RXP on the other hand is defined as SPI_IER_RXPIE, so is intended to be used or the IER register, not the SR register.
Can't you better use SPI_SR_RXP?

Reply:
You are totally right! Thanks, I'm fixing it.